components/dashboard/src/Login.tsx

```diff
@@ -41,52 +40,54 @@ export function Login({ gitpodService }: { gitpodService: GitpodService }) {
     }
 
     return (<div id="login-container" className="z-50 flex w-screen h-screen">
-        <div id="feature-section" className="flex-grow bg-gray-100 w-1/2">
-            <div className="flex flex-col space-y-8 h-full justify-center p-20">
-                <div className="flex space-x-2 text-center">
-                    <img src="/gitpod.svg" className="h-6 my-1" />
-                    <h3>Gitpod</h3>
-                </div>
-                <div>
-                    <h1>Save time<br/> with Prebuilds</h1>
-                </div>
-                <div className="text-gray-400">
-                    Gitpod continuously builds your git branches like a CI server. This means no more waiting for dependencies to be downloaded and builds to finish. Learn more about Prebuilds
-                </div>
-                <div>
-                    {/* pic */}
-                </div>
-            </div>
-        </div>
-        <div id="login-section" className="flex-grow flex flex-col w-1/2">
-
-            <div className="flex-grow h-100 flex flex-row items-center justify-center" >
-                
-                <div className="rounded-xl px-10 py-10 mx-auto">
-                    <div className="mx-auto pb-8">
-                        <img src="/gitpod.svg" className="h-16 mx-auto" />
+        <div id="feature-section" className="flex-grow bg-gray-200 w-1/2">
+            <div id="feature-section-column" className="flex max-w-2xl h-full ml-auto ">
+                <div className="flex flex-col space-y-8 pl-6 pr-24 m-auto">
+                    <div>
+                        <img src="/images/gitpod.svg" className="h-8" />
                     </div>
-                    <div className="mx-auto text-center pb-8 space-y-2">
-                        <h1 className="text-3xl">Log in to Gitpod</h1>
-                        <h2 className="uppercase">ALWAYS READY-TO-CODE</h2>
+                    <div>
+                        <h1 className="text-7xl">Save time<br /> with Prebuilds</h1>
```

Comment:
**nitpick**: Space between logo, heading, paragaph, and the terminal graphic should be 48px. Default spacing for 6 is 48px or should we also customize this? I've added a [design spec](https://www.figma.com/file/cwb0j5H4XVJ8WaZFqIP9bb/Login-Page-3366?node-id=0%3A1) for this. Minor and can be a follow up issue.